docs/readmes/orc8r/deploy_faq.md

```diff
@@ -0,0 +1,138 @@
+---
+id: deploy_faq
+title: FAQs
+hide_title: true
+---
+
+# Deployment FAQs
+
+## Commonly seen issues with Terraform
+
+### Timed out when running “terraform apply”
+
+https://magmacore.slack.com/archives/C018J8UMGMR/p1599228643121500
+
+```
+Error: rpc error: code = Unknown desc = release orc8r failed: timed out waiting for the condition on ../../main.tf line 22, in resource "helm_release" "orc8r":
+ 22: resource "helm_release" "orc8r" {
+```
+
+**Resolution Steps**
+
+* Check if there is an issue with helm repo url, password or container image tag
+* Ensure container image registry and Terraform values are correct
+
+* Run following kubectl commands to get more details on the error encountered
+
+```
+kubectl -n orc8r get pods
+kubectl -n orc8r describe pods
+# pod status ImagePullBackOff, indicates that Image wasn't found
```

Comment:
I think it's also worth noting that a pod could be crash looping. I've often seen this cause timeouts.